src/main/java/com/example/TicketBookingApp/domain/Reservation.java

```diff
@@ -0,0 +1,48 @@
+package com.example.TicketBookingApp.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.List;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder(toBuilder = true)
+public class Reservation {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    private LocalDate date;
+
+    @NotNull
+    private LocalTime time;
+
+    @NotEmpty
+    private String personName;
+
+    @NotEmpty
+    private String personSurname;
+
+    private LocalDateTime expirationTime;
+
+    private Float totalAmountToPay;
+
+    @OneToOne
+    private Screening screening;
```

Comment:
To jest one to one? Jedna rezerwacja na jeden seans może iść?

Reply:
Raczej w drugą stronę - jeden seans może mieć wiele rezerwacji - ba, bez tego to nie ma raczej sensu ;)

Reply:
Fakt, gdyby seans miał jedną rezerwację, byłby to pokaz indywidualny. Zgadza się, słuszna uwaga, dzięki :)